### PR TITLE
finished the code challenge

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401-challenges/src/test/java/code401/challenges/FizzBuzzTree/FizzBuzzTreeTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package code401.challenges.FizzBuzzTree;
+
+import code401.challenges.tree.Node;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class FizzBuzzTreeTest {
```
Comment (evanslaton) [inline code401-challenges/src/test/java/code401/challenges/FizzBuzzTree/FizzBuzzTreeTest.java:8]: It would have been a good idea to test and make sure the your method worked on the entire tree, not just on the root node!
